docs/tutorials/resolution_functions.ipynb

```diff
@@ -0,0 +1,380 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "a60117e3-d089-4375-ac7c-12a52ed47271",
+   "metadata": {},
+   "source": [
+    "# Resolution functions\n",
+    "The purpose of the resolution function is to enable the `EasyReflectometry` model to account for various non ideal effects that might be present in the experimental setup.  When determining reflectivity the resolution function defines the smearing to apply.  For a given Q point such smearing is applied by determining an average of the neighboring Q point weigthed by a normal distribution, which has a Q point dependent Full Width at the Half Maximum (FWHM) that again is defined by the resolution function.\n",
```

Comment:
"model to account for various non ideal effects that might be present in the experimental setup" --> "model to quantify the experimental uncertainties in wavelength and incident angle."

Reply:
done